chore: fix customer return

LeonelCeballos18 opened the pull request:
## What kind of PR is this?
- [x] Bug fix

## Description
-Fix customer return
-changed names in appointment services and modules
-change names variables in dtos

LeonelCeballos18 committed:
chore: fix customer return

cesarj45 commented on src/dtos/appointment.register.dto.ts:
```diff
@@ -10,17 +10,17 @@ export default class AppointmentRegisterDto {
 
   @IsNumber({}, { each: true })
-  service_id: number[];
+  treatment_id: number[];
```
I think this has to be in plural `treatment_ids` would be better.

cesarj45 commented on src/entities/appointment.entity.ts:
```diff
@@ -8,7 +8,7 @@ import {
 } from 'typeorm';
 import { Customer, User, Treatment } from './index';
-import { AppointmentStatus } from '../enums/appointments.status.enum';
+import { status } from '../enums/appointments.status.enum';
```
You forgot to upload the `appointments.status.enum.ts` file, so because of this your branch is crashing.

cesarj45 commented on src/entities/appointment.entity.ts:
```diff
@@ -32,6 +32,12 @@ export default class Appointment {
   @Column({ nullable: false })
   duration: number;
 
+  @Column({ nullable: false })
```
You should use the typeORM decorators for these two columns as we agreed before.
```suggestion
   @CreateDateColumn()
  created_at: Date;

  @UpdateDateColumn()
  updated_at: Date;
```

cesarj45 commented on src/services/appointment.service.ts:
```diff
@@ -79,15 +79,19 @@ export default class AppointmentService {
-      customer: existingCustomer,
+      customer: newCustomer,
       treatments: treatments,
+      created_at: new Date(),
```
If you use decorators, there is no need to set the values, typeORM will do it for you.

cmoreno357 replied:
This should be the case. But there's still some teams that keep it explicit for convention and style. Up to you guys

cesarj45 commented on src/services/appointment.service.ts:
```diff
@@ -79,15 +79,19 @@ export default class AppointmentService {
+      update_at: new Date(),
     });
 
+    console.log(appointment);
```
You should remove this c:

LeonelCeballos18 committed:
chore: Solving comments

-fix `created_at` and `updated_at` fields using decorators
-add migrations
-fix missing objects in appointment return

LeonelCeballos18 committed:
chore: Solving comments
-pluralize treatment_id to treatment_ids in dto file
-change enum class name from `status` to `Status`

cesarj45 reviewed (approved):
LGTM
